flycheck.el

```diff
@@ -9483,7 +9483,7 @@ or if the current buffer has no file name."
 (defun flycheck-rust-cargo-metadata ()
   "Run 'cargo metadata' and return the result as parsed JSON object.
 
-Return nil if the command failed or if a parse error occured."
+We don't ignore any errors in this function."
```

Comment:
I don't think we need to spell out that we don't do any error catching, as that is a detail of implementation.  Anyone caring about implementation can see in the code that no error checking is done.

Please remove this line :)